docs/cookbook/dateTimeFromLegacyDate.mjs

```diff
@@ -0,0 +1,20 @@
+import assert from "assert";
+
+/**
+ * Converts a Date instance into a Temporal.DateTime instance
+ * DateTime accepts an ISO String in its from() method, so if you need to convert a date to a datetime
+ * you're best off passing the result of that
+ *
+ * @param {Date} LegacyDate This is a Date instance
```

Comment:
For consistency with other examples, I'd make this parameter camel case, `legacyDate`.